src/index.html

```diff
@@ -13,6 +13,38 @@
     />
```

Comment:
You should not link a SCSS file in your HTML. Instead, compile your SCSS to CSS and link the resulting CSS file. This is because browsers do not understand SCSS syntax.